docs/running-muse-example.ipynb

```diff
@@ -4,137 +4,143 @@
+    "python -m jupyter jupyter notebook\n",
     "```\n",
     "\n",
-    "A web browser should now open up with a URL such as the following: ```http://localhost:8888/tree```. If it doesn't, copy and paste the command as directed in the Anaconda Powershell Prompt. This will likely take the form of:\n",
+    "A web browser should now open up with a URL such as the following: ```http://localhost:8888/tree```. If it doesn't, copy and paste the command as directed in the terminal. This will likely take the form of:\n",
     "\n",
     "```bash\n",
     "http://localhost:8888/?token=xxxxxxxxxx\n",
     "```\n",
+    "With `xxxxxxxxxx` a very long collection of letters and numbers. Once you are on the page, you will be able to navigate to a location of your choice and create a new file, by clicking the ```new``` button in the top right, followed by clicking the ```Python 3``` button. You should then be able to proceed and follow the tutorials in this documentation.\n",
     "\n",
-    "Once you are on the page, you will be able to navigate to a location of your choice and create a new file, by clicking the ```new``` button in the top right, followed by clicking the ```Python 3``` button.\n",
-    "\n",
-    "You should then be able to proceed and follow the tutorials in this documentation.\n",
+    "### Missing packages\n",
     "\n",
-    "If you get any errors such as:\n",
+    "If when running a cell you get any errors such as:\n",
     "\n",
     "```bash\n",
     "ModuleNotFoundError: No module named 'pandas'\n",
     "```\n",
     "\n",
-    "It is possible to install the required packages by running the following in the Anaconda Powershell Prompt:\n",
+    "Then you are trying to use a package (`pandas` in the example) that is not available in the current environment. It is possible to install the missing packages by running the following in the jupyter notebook:\n",
     "\n",
     "```bash\n",
-    "conda install pandas\n",
+    "!pip install pandas\n",
     "```\n",
     "\n",
-    "or running the following in jupyter notebook:\n",
+    "The package will be installed in whatever virtual environment Jupyter is running into."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Results\n",
     "\n",
-    "```bash\n",
-    "!conda install pandas\n",
-    "```"
+    "If the default MUSE example has run successfully, you should now have a folder called ```Results``` in the current working directory.\n",
+    "\n",
+    "This directory should contain results for each sector (```Gas```,```Power``` and ```Residential```) as well as results for the entire simulation in the form of ```MCACapacity.csv``` and ```MCAPrices.csv```.\n",
+    "\n",
+    "- ```MCACapacity.csv``` contains information about the capacity each agent has per technology per benchmark year. Each benchmark year is the modelled year in the `settings.toml` file. In our example, this is 2020, 2025, ..., 2050.\n",
+    "\n",
```

Comment:
Multiple rows for every combination of agent-technoloy-region-year in the `MCACapacity.csv' represents capacity of single assets.